package.json

```diff
@@ -1,22 +1,29 @@
 {
   "name": "terriajs-server",
-  "version": "3.3.0",
+  "version": "3.3.1",
   "description": "NodeJS server for TerriaJS, consisting of a CORS proxy, proj4 CRS lookup service, ogr2ogr conversion service, and express static server.",
   "engineStrict": true,
   "engines": {
     "node": ">=8.0.0"
   },
-  "main": "lib/app.js",
+  "main": "dist/app.js",
+  "files": [
+    "/dist"
+  ],
   "scripts": {
-    "test": "jasmine",
+    "lint": "eslint lib/**/*.ts lib/**/*.js spec/**/*.ts",
+    "build": "tsc --module commonjs ",
```

Comment:
the `--module` flag shouldn't be necessary here because you've specified it in your tsconfig.